yt_dlp/extractor/nts.py

```diff
@@ -0,0 +1,76 @@
+from .common import InfoExtractor
+from ..utils import parse_iso8601, url_or_none
+from ..utils.traversal import traverse_obj
+
+
+class NTSLiveIE(InfoExtractor):
+    IE_NAME = 'nts.live'
+    _VALID_URL = r'https?://(?:www\.)?nts\.live/shows/[^/]+/episodes/(?P<id>[^./?#]+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?nts\.live/shows/[^/?#]+/episodes/(?P<id>[^./?#]+)'
```

btw why `[^.]`?

Reply:
> btw why `[^.]`?

I might've been using another extractor as a template, it's true that it's unlikely to make a difference.